ui/src/main/webapp/app/gaffer/query-service.js

```diff
@@ -16,11 +16,31 @@
 
 'use strict';
 
-angular.module('app').factory('query', ['$http', 'config', 'events', 'common', 'error', 'loading', 'settings', 'results', '$mdDialog', function($http, config, events, common, error, loading, settings, results, $mdDialog) {
+angular.module('app').factory('query', ['$http', 'config', 'events', 'common', 'error', 'loading', 'settings', 'results', '$mdDialog', '$routeParams', '$location', function($http, config, events, common, error, loading, settings, results, $mdDialog, $routeParams, $location) {
 
     var query = {};
     var operations = [];
 
+    query.executeOperationJson = function(operationJson) {
+        if(operationJson) {
+            var operationJsonArray;
+            if(Array.isArray(operationJson)) {
+                operationJsonArray = operationJson;
+            } else {
+                operationJsonArray = [operationJson];
+            }
+            for (var i in operationJsonArray) {
+                try {
+                    var operation = JSON.parse(operationJsonArray[i]);
+                    query.addOperation(operation);
+                    query.executeQuery(operation);
+                } catch {
```

Comment:
```suggestion
                } catch (err) {
```
This will fix error on Travis CI